src/Lachain.Core/RPC/HTTP/Web3/TransactionServiceWeb3.cs

```diff
@@ -475,6 +475,19 @@ public string Call(JObject opts, string? blockId)
                 InvocationResult invRes = _stateManager.SafeContext(() =>
                 {
                     var snapshot = _stateManager.NewSnapshot();
+					if (!tx.Value.IsZero())
+					{
+                        var transferContext = new InvocationContext(source, snapshot, new TransactionReceipt
+                        {
+                            Block = snapshot.Blocks.GetTotalBlockHeight(),
+                            Transaction = tx
+                        });
+                    
+                        var localInvocation = ContractEncoder.Encode("transfer(address,uint256)", destination, tx.Value);
+                        var transferResult =
+                            ContractInvoker.Invoke(ContractRegisterer.LatokenContract, transferContext, localInvocation, GasMetering.DefaultBlockGasLimit);
+                        gasUsed += transferResult.GasUsed;
```

Comment:
@sgladkov Should we rollback stateManager here using `_stateManager.Rollback();`?

Reply:
No, we make rollback after the script execution. The goal of this fix is to make transferred funds available to the contract during its execution emulation